test/topology_experimental_raft/test_tablets.py

```diff
@@ -1296,3 +1315,186 @@ async def test_tablet_storage_freeing(manager: ManagerClient):
     logger.info("Verify that the table's disk usage on first node shrunk by about half.")
     size_after = await manager.server_get_sstables_disk_usage(servers[0].server_id, "test", "test")
     assert size_before * 0.33 < size_after < size_before * 0.66
+
+
+async def create_cluster(manager: ManagerClient, num_dcs: int, num_racks: int, nodes_per_rack: int) -> dict[ServerNum, ServerInfo]:
+    logger.debug(f"Creating cluster: num_dcs={num_dcs} num_racks={num_racks} nodes_per_rack={nodes_per_rack}")
+    servers: dict[ServerNum, ServerInfo] = dict()
+    for dc in range(1, num_dcs + 1):
+        for rack in range(1, num_racks + 1):
+            rack_servers = await manager.servers_add(nodes_per_rack, property_file={"dc": f"dc{dc}", "rack": f"rack{rack}"})
+            for s in rack_servers:
+                servers[s.server_id] = s
+    logger.debug(f"Created servers={list(servers.values())}")
+    return servers
+
+
+async def create_and_populate_table(manager: ManagerClient, ks: str = "ks", table: str = "test", rf: int = 3, initial_tablets: int = 64, num_keys: int = 0) -> tuple[str, str, int, int]:
+    logger.info(f"Creating table and populating data {ks}.{table}: rf={rf} initial_tablets={initial_tablets} num_keys={num_keys}")
+    cql = manager.get_cql()
+    await cql.run_async(f"CREATE KEYSPACE {ks} WITH replication = {{'class': 'NetworkTopologyStrategy', 'replication_factor': {rf}}} AND tablets = {{'initial': {initial_tablets}}};")
+    await cql.run_async(f"CREATE TABLE {ks}.{table} (pk int PRIMARY KEY, c int);")
+
+    if not num_keys:
+        num_keys = initial_tablets * 4
+
+    await asyncio.gather(*[cql.run_async(f"INSERT INTO {ks}.{table} (pk, c) VALUES ({k}, 1);") for k in range(num_keys)])
+
+    return (ks, table, initial_tablets, num_keys)
```

Comment:
no need for the brackets

Reply:
well, they're parenthesis ;)
IMO it better represents that a tuple is being returned, but I speak Python only as a second (or third, forth, don't remember) language :)

Reply:
Yeah, it is a very minor nitpick, both are fine, without parentheses is little bit more common. Feel free to ignore